[Bug]: image to pdf not work while merge into one PDF

### Installation Method

Docker

### The Problem

![image](https://github.com/user-attachments/assets/9d2b79ab-1eca-45b4-9165-ac5d1992576c)
if I choose two images and then select merge into one PDF. It doesn't work. Only comes one image in PDF
![image](https://github.com/user-attachments/assets/5a271148-be44-41a9-9075-30ad479c0d82)


### Version of Stirling-PDF

App Version: 0.36.2  github docker

### Last Working Version of Stirling-PDF

_No response_

### Page Where the Problem Occurred

https://nas.mowen.store:8584/img-to-pdf?lang=en_US

### Docker Configuration

_No response_

### Relevant Log Output

```txt
date	stream	content
2024/12/14 23:56:52	stdout	15:56:52.093 [qtp1422140849-48] INFO  s.software.SPDF.utils.PdfUtils - PDF successfully saved to byte array
2024/12/14 23:56:09	stdout	... 53 common frames omitted
2024/12/14 23:56:09	stdout	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1014)
2024/12/14 23:56:09	stdout	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:978)
2024/12/14 23:56:09	stdout	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1088)
2024/12/14 23:56:09	stdout	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
2024/12/14 23:56:09	stdout	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:891)
2024/12/14 23:56:09	stdout	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:986)
2024/12/14 23:56:09	stdout	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:118)
2024/12/14 23:56:09	stdout	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:188)
2024/12/14 23:56:09	stdout	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:255)
2024/12/14 23:56:09	stdout	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
2024/12/14 23:56:09	stdout	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
2024/12/14 23:56:09	stdout	at stirling.software.SPDF.controller.api.converters.ConvertImgPDFController.convertToPdf(ConvertImgPDFController.java:216)
2024/12/14 23:56:09	stdout	at stirling.software.SPDF.utils.PdfUtils.imageToPdf(PdfUtils.java:413)
2024/12/14 23:56:09	stdout	at stirling.software.SPDF.utils.ImageProcessingUtils.convertColorType(ImageProcessingUtils.java:31)
2024/12/14 23:56:09	stdout	Caused by: java.lang.NullPointerException: Cannot invoke "String.hashCode()" because "<local3>" is null
2024/12/14 23:56:09	stdout	at java.base/java.lang.Thread.run(Thread.java:1583)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1164)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.doRunJob(QueuedThreadPool.java:1209)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:979)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:311)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.run(AdaptiveExecutionStrategy.java:201)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.tryProduce(AdaptiveExecutionStrategy.java:293)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.consumeTask(AdaptiveExecutionStrategy.java:441)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.runTask(AdaptiveExecutionStrategy.java:478)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.io.SelectableChannelEndPoint$1.run(SelectableChannelEndPoint.java:53)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:99)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:322)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.server.internal.HttpConnection.onFillable(HttpConnection.java:418)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.server.internal.HttpChannelState$HandlerInvoker.run(HttpChannelState.java:662)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.server.Server.handle(Server.java:182)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.server.handler.EventsHandler.handle(EventsHandler.java:81)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.server.Handler$Wrapper.handle(Handler.java:740)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.server.handler.ContextHandler.handle(ContextHandler.java:1060)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.SessionHandler.handle(SessionHandler.java:717)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:575)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler.handle(ServletHandler.java:464)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletChannel.handle(ServletChannel.java:436)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletChannel.dispatch(ServletChannel.java:819)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$MappedServlet.handle(ServletHandler.java:1547)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:56:09	stdout	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
2024/12/14 23:56:09	stdout	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:56:09	stdout	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
2024/12/14 23:56:09	stdout	at org.springframework.web.filter.ServerHttpObservationFilter.doFilterInternal(ServerHttpObservationFilter.java:114)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:56:09	stdout	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
2024/12/14 23:56:09	stdout	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:56:09	stdout	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
2024/12/14 23:56:09	stdout	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:56:09	stdout	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
2024/12/14 23:56:09	stdout	at stirling.software.SPDF.config.MetricsFilter.doFilterInternal(MetricsFilter.java:48)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.websocket.servlet.WebSocketUpgradeFilter.doFilter(WebSocketUpgradeFilter.java:195)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$ChainEnd.doFilter(ServletHandler.java:1614)
2024/12/14 23:56:09	stdout	at org.eclipse.jetty.ee10.servlet.ServletHolder.handle(ServletHolder.java:736)
2024/12/14 23:56:09	stdout	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:614)
2024/12/14 23:56:09	stdout	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:885)
2024/12/14 23:56:09	stdout	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:547)
2024/12/14 23:56:09	stdout	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:914)
2024/12/14 23:56:09	stdout	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1022)
2024/12/14 23:56:09	stdout	jakarta.servlet.ServletException: Request processing failed: java.lang.NullPointerException: Cannot invoke "String.hashCode()" because "<local3>" is null
2024/12/14 23:56:09	stdout	15:56:09.555 [qtp1422140849-70] WARN  o.e.j.ee10.servlet.ServletChannel - /api/v1/convert/img/pdf
2024/12/14 23:55:57	stdout	... 53 common frames omitted
2024/12/14 23:55:57	stdout	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1014)
2024/12/14 23:55:57	stdout	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:978)
2024/12/14 23:55:57	stdout	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1088)
2024/12/14 23:55:57	stdout	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
2024/12/14 23:55:57	stdout	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:891)
2024/12/14 23:55:57	stdout	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:986)
2024/12/14 23:55:57	stdout	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:118)
2024/12/14 23:55:57	stdout	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:188)
2024/12/14 23:55:57	stdout	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:255)
2024/12/14 23:55:57	stdout	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
2024/12/14 23:55:57	stdout	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
2024/12/14 23:55:57	stdout	at stirling.software.SPDF.controller.api.converters.ConvertImgPDFController.convertToPdf(ConvertImgPDFController.java:216)
2024/12/14 23:55:57	stdout	at stirling.software.SPDF.utils.PdfUtils.imageToPdf(PdfUtils.java:413)
2024/12/14 23:55:57	stdout	at stirling.software.SPDF.utils.ImageProcessingUtils.convertColorType(ImageProcessingUtils.java:31)
2024/12/14 23:55:57	stdout	Caused by: java.lang.NullPointerException: Cannot invoke "String.hashCode()" because "<local3>" is null
2024/12/14 23:55:57	stdout	at java.base/java.lang.Thread.run(Thread.java:1583)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1164)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.doRunJob(QueuedThreadPool.java:1209)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:979)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:311)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.run(AdaptiveExecutionStrategy.java:201)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.tryProduce(AdaptiveExecutionStrategy.java:293)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.consumeTask(AdaptiveExecutionStrategy.java:441)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.runTask(AdaptiveExecutionStrategy.java:478)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.io.SelectableChannelEndPoint$1.run(SelectableChannelEndPoint.java:53)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:99)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:322)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.server.internal.HttpConnection.onFillable(HttpConnection.java:418)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.server.internal.HttpChannelState$HandlerInvoker.run(HttpChannelState.java:662)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.server.Server.handle(Server.java:182)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.server.handler.EventsHandler.handle(EventsHandler.java:81)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.server.Handler$Wrapper.handle(Handler.java:740)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.server.handler.ContextHandler.handle(ContextHandler.java:1060)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.SessionHandler.handle(SessionHandler.java:717)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:575)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler.handle(ServletHandler.java:464)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletChannel.handle(ServletChannel.java:436)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletChannel.dispatch(ServletChannel.java:819)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$MappedServlet.handle(ServletHandler.java:1547)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:55:57	stdout	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
2024/12/14 23:55:57	stdout	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:55:57	stdout	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
2024/12/14 23:55:57	stdout	at org.springframework.web.filter.ServerHttpObservationFilter.doFilterInternal(ServerHttpObservationFilter.java:114)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:55:57	stdout	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
2024/12/14 23:55:57	stdout	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:55:57	stdout	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
2024/12/14 23:55:57	stdout	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:55:57	stdout	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
2024/12/14 23:55:57	stdout	at stirling.software.SPDF.config.MetricsFilter.doFilterInternal(MetricsFilter.java:48)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1586)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.FilterHolder.doFilter(FilterHolder.java:205)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.websocket.servlet.WebSocketUpgradeFilter.doFilter(WebSocketUpgradeFilter.java:195)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletHandler$ChainEnd.doFilter(ServletHandler.java:1614)
2024/12/14 23:55:57	stdout	at org.eclipse.jetty.ee10.servlet.ServletHolder.handle(ServletHolder.java:736)
2024/12/14 23:55:57	stdout	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:614)
2024/12/14 23:55:57	stdout	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:885)
2024/12/14 23:55:57	stdout	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:547)
2024/12/14 23:55:57	stdout	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:914)
2024/12/14 23:55:57	stdout	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1022)
2024/12/14 23:55:57	stdout	jakarta.servlet.ServletException: Request processing failed: java.lang.NullPointerException: Cannot invoke "String.hashCode()" because "<local3>" is null
2024/12/14 23:55:57	stdout	15:55:57.279 [qtp1422140849-58] WARN  o.e.j.ee10.servlet.ServletChannel - /api/v1/convert/img/pdf
2024/12/14 23:55:38	stdout	15:55:38.650 [qtp1422140849-34] INFO  s.software.SPDF.utils.PdfUtils - PDF successfully saved to byte array
2024/12/14 23:54:52	stdout	15:54:52.795 [qtp1422140849-58] INFO  s.software.SPDF.utils.PdfUtils - PDF successfully saved to byte array
2024/12/14 23:52:37	stdout	15:52:37.380 [qtp1422140849-34] INFO  s.software.SPDF.utils.PdfUtils - PDF successfully saved to byte array
2024/12/14 23:49:46	stdout	15:49:46.626 [qtp1422140849-28] INFO  o.s.api.AbstractOpenApiResource - Init duration for springdoc-openapi is: 1956 ms
2024/12/14 20:53:25	stdout	12:53:25.408 [main] INFO  s.software.SPDF.SPdfApplication - Navigate to http://localhost:8080
2024/12/14 20:53:25	stdout	12:53:25.407 [main] INFO  s.software.SPDF.SPdfApplication - Stirling-PDF Started.
2024/12/14 20:53:25	stdout	12:53:25.405 [scheduling-1] INFO  s.software.SPDF.utils.FileMonitor - Registered directory: ./pipeline/watchedFolders
2024/12/14 20:53:25	stdout	12:53:25.403 [scheduling-1] WARN  s.software.SPDF.utils.FileMonitor - not monitoring any directory, even the root directory itself: ./pipeline/watchedFolders
2024/12/14 20:53:25	stdout	12:53:25.388 [main] INFO  s.software.SPDF.SPdfApplication - Started SPdfApplication in 21.838 seconds (process running for 29.197)
2024/12/14 20:53:20	stdout	12:53:20.368 [main] INFO  s.s.S.config.EndpointConfiguration - Total disabled endpoints: 3. Disabled endpoints: book-to-pdf, pdf-to-book, pdf-to-pdfa
2024/12/14 20:53:13	stdout	12:53:13.962 [main] INFO  s.software.SPDF.SPdfApplication - Running configs ApplicationProperties(legal=ApplicationProperties.Legal(termsAndConditions=https://www.stirlingpdf.com/terms-and-conditions, privacyPolicy=https://www.stirlingpdf.com/privacy-policy, accessibilityStatement=, cookiePolicy=, impressum=), security=ApplicationProperties.Security(enableLogin=false, csrfDisabled=false, initialLogin=ApplicationProperties.Security.InitialLogin(username=), oauth2=ApplicationProperties.Security.OAUTH2(enabled=false, issuer=, clientId=, autoCreateUser=true, blockRegistration=false, useAsUsername=email, scopes=[openid, profile, email], provider=google, client=ApplicationProperties.Security.OAUTH2.Client(google=Google [clientId=, clientSecret=NULL, scopes=[https://www.googleapis.com/auth/userinfo.email, https://www.googleapis.com/auth/userinfo.profile], useAsUsername=email], github=GitHub [clientId=, clientSecret=NULL, scopes=[read:user], useAsUsername=login], keycloak=Keycloak [issuer=, clientId=, clientSecret=NULL, scopes=[openid, profile, email], useAsUsername=preferred_username])), saml2=stirling.software.SPDF.model.ApplicationProperties$Security$SAML2@11e834ad, loginAttemptCount=5, loginResetTimeMinutes=120, loginMethod=all, customGlobalAPIKey=null), system=ApplicationProperties.System(defaultLocale=en-US, googlevisibility=false, showUpdate=false, showUpdateOnlyAdmin=false, customHTMLFiles=false, tessdataDir=/usr/share/tessdata, enableAlphaFunctionality=false, enableAnalytics=undefined), ui=ApplicationProperties.Ui(appName=null, homeDescription=null, appNameNavbar=null), endpoints=ApplicationProperties.Endpoints(toRemove=[], groupsToRemove=[]), metrics=ApplicationProperties.Metrics(enabled=true), automaticallyGenerated=ApplicationProperties.AutomaticallyGenerated(UUID=example, appVersion=0.35.0), enterpriseEdition=ApplicationProperties.EnterpriseEdition(enabled=false, maxUsers=0, customMetadata=ApplicationProperties.EnterpriseEdition.CustomMetadata(autoUpdateMetadata=false, author=username, creator=Stirling-PDF, producer=Stirling-PDF)), autoPipeline=ApplicationProperties.AutoPipeline(outputFolder=null), processExecutor=ApplicationProperties.ProcessExecutor(sessionLimit=ApplicationProperties.ProcessExecutor.SessionLimit(libreOfficeSessionLimit=1, pdfToHtmlSessionLimit=1, pythonOpenCvSessionLimit=8, weasyPrintSessionLimit=16, installAppSessionLimit=1, calibreSessionLimit=1, qpdfSessionLimit=4, tesseractSessionLimit=1), timeoutMinutes=ApplicationProperties.ProcessExecutor.TimeoutMinutes(libreOfficeTimeoutMinutes=30, pdfToHtmlTimeoutMinutes=20, pythonOpenCvTimeoutMinutes=30, weasyPrintTimeoutMinutes=30, installAppTimeoutMinutes=60, calibreTimeoutMinutes=30, tesseractTimeoutMinutes=30, qpdfTimeoutMinutes=30)))
2024/12/14 20:53:06	stdout	12:53:06.430 [main] INFO  s.software.SPDF.SPdfApplication - The following 1 profile is active: "default"
2024/12/14 20:53:06	stdout	12:53:06.413 [main] INFO  s.software.SPDF.SPdfApplication - Starting SPdfApplication v0.36.2 using Java 21.0.5 with PID 11 (/app.jar started by stirlingpdfuser in /)
2024/12/14 20:53:06	stdout	Powered by Spring Boot 3.4.0
2024/12/14 20:53:06	stdout	|____/ |_| |___|_| \_\_____|___|_| \_|\____|     |_|   |____/|_|
2024/12/14 20:53:06	stdout	 ___) || |  | ||  _ <| |___ | || |\  | |_| |_____|  __/| |_| |  _|
2024/12/14 20:53:06	stdout	\___ \ | |  | || |_) | |    | ||  \| | |  _ _____| |_) | | | | |_
2024/12/14 20:53:06	stdout	/ ___|_   _|_ _|  _ \| |   |_ _| \ | |/ ___|     |  _ \|  _ \|  ___|
2024/12/14 20:53:06	stdout	 ____ _____ ___ ____  _     ___ _   _  ____       ____  ____  _____
2024/12/14 20:53:05	stdout	12:53:03.544 [main] WARN  s.software.SPDF.SPdfApplication - Custom configuration file 'configs/custom_settings.yml' does not exist.
2024/12/14 20:53:03	stdout	12:53:03.540 [main] WARN  s.software.SPDF.SPdfApplication - External configuration file 'configs/settings.yml' does not exist.
2024/12/14 20:52:56	stderr	Picked up JAVA_TOOL_OPTIONS:  -XX:MaxRAMPercentage=75
2024/12/14 20:52:55	stdout	Setting permissions and ownership for necessary directories...
2024/12/14 20:52:55	stdout	Running Stirling PDF with DOCKER_ENABLE_SECURITY=false and VERSION_TAG=0.36.2
2024/12/14 20:52:45	stdout	Copying original files without overwriting existing files
```


### Additional Information

_No response_

### Browsers Affected

Chrome

### No Duplicate of the Issue

- [X] I have verified that there are no existing issues raised related to my problem.

--- a/src/main/resources/static/js/downloader.js
+++ b/src/main/resources/static/js/downloader.js
@@ -75,9 +75,10 @@
           // Check if any PDF files are encrypted and handle decryption if necessary
           const decryptedFiles = await checkAndDecryptFiles(url, files);
           files = decryptedFiles;
+          formData.delete('fileInput'); // Reset fileInput and Append
           // Append decrypted files to formData
           decryptedFiles.forEach((file, index) => {
-            formData.set(`fileInput`, file);
+            formData.append(`fileInput`, file);
           });
         }
 

--- a/src/main/resources/templates/convert/img-to-pdf.html
+++ b/src/main/resources/templates/convert/img-to-pdf.html
@@ -51,15 +51,11 @@
                 <br>
                 <button type="submit" id="submitBtn" class="btn btn-primary" th:text="#{imageToPDF.submit}"></button>
                 <script>
-                  $('#fileInput-input').on('change', function() {
+                    $('#fileInput-input').on('file-input-change', () => {
                     var files = document.getElementById("fileInput-input").files;
                     var conversionType = document.getElementById("conversionType");
                     console.log("files.length=" + files.length)
-                    if (files.length > 1) {
-                      conversionType.disabled = false;
-                    } else {
-                      conversionType.disabled = true;
-                    }
+                    conversionType.disabled = files.length <= 1;
                   });
 
                   $('#conversionType').change(function() {

